build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 subprojects {
 
     group = 'edu.wisc.my.restproxy'
-    version = '3.0.1'
+    version = '3.0.3'
```

Comment:
Pedantic: adds a new feature, so SemVer as I understand it would prescribe `3.1.0`.